apps/src/aichat/views/chatWorkspace.module.scss

```diff
@@ -12,6 +12,7 @@ $default-spacing: 2px;
 
 .conversationArea {
   margin: 5px 2%;
+  height: 65vh;
```

Comment:
Added so that chat workspace is scrollable.

Reply:
Style update moved to separate PR https://github.com/code-dot-org/code-dot-org/pull/57996